semgrep_output_v1.atd

```diff
@@ -1803,6 +1803,7 @@ type function_call
   | CallApplyFixes of apply_fixes_params
   | CallSarifFormat of sarif_format_params
   | CallFormatter of (output_format * cli_output)
+  | CallValidate of fpath
```

Comment:
please add the comment to say that it's usually a temporary file containing all the rules, probably with JSON
content (we could insteadt have send the rules via a big string through the RPC pipe)